makefile

```diff
@@ -22,33 +22,30 @@ CFLAGS   = $(INCS) -m64 -Wall -pedantic -g3
 RM       = rm -f
 
 ENABLE_OPENCL ?= 1
+ENABLE_COMPLEX_X2 ?= 1
```

Comment:
Gets removed in favor of whatever is in the config.h

Reply:
Done.